test/Microsoft.ML.Tests/TrainerEstimators/FAFMEstimator.cs

```diff
@@ -7,14 +7,15 @@
 using Microsoft.ML.Data;
 using Microsoft.ML.RunTests;
 using Microsoft.ML.TestFrameworkCommon;
+using Microsoft.ML.TestFramework.Attributes;
 using Microsoft.ML.Trainers;
 using Xunit;
 
 namespace Microsoft.ML.Tests.TrainerEstimators
 {
     public partial class TrainerEstimators : TestDataPipeBase
     {
-        [Fact]
+        [NativeDependencyFact("CpuMathNative")]
```

Comment:
This doesn't feel right. I know I said this elsewhere, and I won't repeat it for all places. Basically I don't think we should have tests that say they can only run with `CpuMathNative`. CpuMathNative is an implementation detail that shouldn't affect how the API behaves.

Reply:
I'll figure out again exactly what part it is, but there are a few parts of CpuMathNative that have not been re-written with a managed fallback yet. The majority has been, but a few methods haven't been. I'll get back to you with more details.

Reply:
> there are a few parts of CpuMathNative that have not been re-written with a managed fallback yet

I wasn't aware of this. If this is true, it needs to be fixed. But I was pretty sure it was completely re-written in managed code.

Reply:
Same comment as above.
> Adding on to my comment below about not all parts of CPU Math having a fallback. This test uses the factorization machine. Here is the relevant code:
https://github.com/dotnet/machinelearning/blob/1b3cb77b9752fe4279376039ee20fc42822e4845/src/Microsoft.ML.CpuMath/FactorizationMachine/FactorizationMachineInterface.netcoreapp.cs#L33-L36
arm doesn't support AVX, and the only other option is to use the native code in CPU math, which uses the x86/x64 SIMD instructions.
I have added an issue, #5871, to track this. IIRC this is the only case in CPU Math that doesn't have a basic managed fallback.